hazelcast/include/hazelcast/client/MembershipEvent.h

```diff
@@ -76,7 +78,7 @@ namespace hazelcast {
              *
              * @return the members at the moment after this event.
              */
-            virtual const std::vector<Member> getMembers() const;
+            virtual const std::set<Member> getMembers() const;
```

Comment:
This is public API. We can not change return value.

Reply:
Ok, reverted back.